_config.yml

```
@@ -1,2 +1,17 @@
-title: Welcome to my portfolio!
-theme: jekyll-theme-cayman
\ No newline at end of file
+title: Portafolio de Olatz Balda
+description: Bienvenido a mi página Soy una estudiante de Comunicación de la Universidad de Deusto. Este es mi repositorio de la asignatura 'Mensajes para la Web'.
```

Falta un punto aquí: 
```suggestion
description: Bienvenido a mi página. Soy una estudiante de Comunicación de la Universidad de Deusto. Este es mi repositorio de la asignatura 'Mensajes para la Web'.
```